atlp-rwanda/ecommerce-app-champions-fe
# #66 chore: Configure storybook for component development

#### What does this PR do?
- configure react storybook
#### Description of Task to be completed
- implement storybook setup
- implement sample components to test with story book
#### How should this be manually tested?
- clone the repo using git clone repo's URL
- change branch git checkout ch-story-book
- create .env file  and paste variables from .env.example
- run `npm run storybook`
- you can see the new stories of the components I created such as `MyButton,MyHeader,HomePage`
#### What are the relevant pivotal tracker stories?
- ![](https://github.trello.services/images/mini-trello-icon.png) [Configure storybook for component development](https://trello.com/c/JdSbypiw)
